plugin/src/main/java/org/opensearch/ml/profile/MLModelProfile.java

```diff
@@ -79,6 +82,9 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         if (memSizeEstimationGPU != null) {
             builder.field("memory_size_estimation_gpu", memSizeEstimationGPU);
         }
+        if (isHidden != null) {
+            builder.field("is_hidden", isHidden);
```

Comment:
Do we need to set `is_hidden: false` for normal model?

Reply:
changed it to:

```
        if (isHidden != null && isHidden) {
            builder.field("is_hidden", true);
        }
        ```